dotcom-rendering/src/types/blocksRequest.ts

```diff
@@ -0,0 +1,21 @@
+import type { BaseRequestBody } from './request';
+
+/**
+ * BlocksRequest is the expected body format for POST requests made to /Blocks
+ */
+export interface FEBlocksRequest extends BaseRequestBody {
+	blocks: Block[];
+	format: FEFormat;
+	host?: string;
+	webTitle: string;
+	ajaxUrl: string;
+	isAdFreeUser: boolean;
+	isSensitive: boolean;
+	edition: string;
+	section: string;
+	sharedAdTargeting: Record<string, unknown>;
+	adUnit: string;
+	videoDuration?: number;
+	switches: { [key: string]: boolean };
+	keywordIds: string;
+}
```

Comment:
Not sure how we do file naming, as a new person to the codebase, I would find it odd to have the file called `blocksRequest` for a `FEBlocksRequest` type.